addOns/spider/src/main/java/org/zaproxy/addon/spider/SpiderParam.java

```diff
@@ -646,6 +652,15 @@ public boolean isParseGit() {
         return parseGit;
     }
 
+    /**
+     * Checks if the spider should parse the >DS_Store files for URIs.
+     *
+     * @return true, if it parses the files
+     */
+    public boolean isParseDsStore() {
```

Comment:
Be good to add tests while this code is not critical the more tests the merrier.